packages/python/chart-studio/chart_studio/tests/test_optional/test_utils/test_utils.py

```diff
@@ -6,7 +6,7 @@
 from datetime import datetime as dt
 import numpy as np
 
-np_list = np.array([1, 2, 3, np.NaN, np.NAN, np.Inf, dt(2014, 1, 5)])
+np_list = np.array([1, 2, 3, np.nan, np.inf, dt(2014, 1, 5)])
```

Comment:
I think we need to have separate tests for both numpy v1 and v2.

Reply:
I think that's a good idea actually!

Reply:
In [#4696](https://github.com/plotly/plotly.py/pull/4696/files) @marthacryan added a CircleCI test for the case when numpy is not installed.
Instead of changing the tests that use `numpy_v1`, what I want to see is to add tests for numpy_v2.